model_templates/triton_onnx_unstructured/client/datarobot-predict.py

```diff
@@ -0,0 +1,147 @@
+"""
+Usage:
+    python datarobot-predict.py <input-file> [mimetype] [charset]
+ 
+This example uses the requests library which you can install with:
+    pip install requests
+We highly recommend that you update SSL certificates with:
+    pip install -U "urllib3[secure]" certifi
+"""
+import sys
+import numpy as np
+from PIL import Image
+from torchvision import transforms
+
+import requests
+import json
+
+# See README.md on how to set up those keys
+API_URL = '<DATAROBOT_API_URL>'
+API_KEY = '<DATAROBOT_API_KEY>'
+DATAROBOT_KEY = '<DATAROBOT_KEY>'
+DEPLOYMENT_ID = '<DEPLOYMENT_ID>'
+
+# Don't change this. It is enforced server-side too.
+MAX_PREDICTION_FILE_SIZE_BYTES = 52428800  # 50 MB
+
+
+class DataRobotPredictionError(Exception):
+    """Raised if there are issues getting predictions from DataRobot"""
+
+
+def make_datarobot_deployment_unstructured_predictions(data, deployment_id, mimetype, charset):
+    """
+    Make unstructured predictions on data provided using DataRobot deployment_id provided.
+    See docs for details:
+         https://docs.datarobot.com/en/docs/api/reference/predapi/dr-predapi.html
+ 
+    Parameters
+    ----------
+    data : bytes
+        Bytes data read from provided file.
+    deployment_id : str
+        The ID of the deployment to make predictions with.
+    mimetype : str
+        Mimetype describing data being sent.
+        If mimetype starts with 'text/' or equal to 'application/json',
+        data will be decoded with provided or default(UTF-8) charset
+        and passed into the 'score_unstructured' hook implemented in custom.py provided with the model.
+ 
+        In case of other mimetype values data is treated as binary and passed without decoding.
+    charset : str
+        Charset should match the contents of the file, if file is text.
+ 
+    Returns
+    -------
+    data : bytes
+        Arbitrary data returned by unstructured model.
+ 
+ 
+    Raises
+    ------
+    DataRobotPredictionError if there are issues getting predictions from DataRobot
+    """
+    # Set HTTP headers. The charset should match the contents of the file.
+    headers = {
+        'Content-Type': '{};charset={}'.format(mimetype, charset),
+        'Authorization': 'Bearer {}'.format(API_KEY),
+        'DataRobot-Key': DATAROBOT_KEY,
+    }
+
+    url = API_URL.format(deployment_id=deployment_id)
+
+    # Make API request for predictions
+    predictions_response = requests.post(
+        url,
+        data=data,
+        headers=headers,
+    )
+    _raise_dataroboterror_for_status(predictions_response)
+    # Return raw response content
+    return predictions_response.content
```

Comment:
Since this script is dedicated to this example and this example has the model checked in, we should know the response of the payload. Shouldn't it just be the classification of the image -- can't we return something more useful?

Reply:
Yeah, that seems like the right thing to do. We may need to load the config.pbtxt file though because that dictates the output? Or do we not need to parse the output that deeply? 

Conversely, just to get us through sign-offs, maybe we can just use `raw_decode()` for now: https://docs.python.org/3/library/json.html#json.JSONDecoder.raw_decode

And in the next iteration, update the predictor?

Reply:
the `raw_decode` solves the problem, thank you for the suggestion

```
from json import JSONDecoder

import numpy as np
import tritonclient.http as httpclient

response_body = b'{"model_name":"densenet_onnx","model_version":"1","outputs":[{"name":"fc6_1","datatype":"BYTES","shape":[1000],"parameters":{"binary_data_size":26754}}]}\x16\x00\x00\x0011.547369:92:BEE EATER\x19\x00\x00\x0011.230166:14:INDIGO FINCH\x13\x00\x00\x007.527154:95:JACAMAR\x0f\x00\x00\x006.921648:17:JAY\x11\x00\x00\x006.576403:88:MACAW\x14\x00\x00\x006.264347:90:LORIKEET\x15\x00\x00\x006.212578:11:GOLDFINCH\x1f\x00\x00\x006.136209:136:EUROPEAN GALLINULE\x15\x00\x00\x006.045386:10:BRAMBLING\x17\x00\x00\x005.643104:94:HUMMINGBIRD\x17\x00\x00\x005.617190:12:HOUSE FINCH\x12\x00\x00\x005.214071:18:MAGPIE\x10\x00\x00\x004.905297:21:KITE\x15\x00\x00\x004.873964:86:PARTRIDGE\x12\x00\x00\x004.588033:96:TOUCAN\x15\x00\x00\x004.586961:723:PINWHEEL\x13\x00\x00\x004.551550:971:BUBBLE\x1e\x00\x00\x004.441406:131:LITTLE BLUE HERON\x11\x00\x00\x004.368442:15:ROBIN\x11\x00\x00\x004.365045:85:QUAIL\x15\x00\x00\x004.204632:19:CHICKADEE\x17\x00\x00\x004.116632:20:WATER OUZEL\x16\x00\x00\x004.014774:319:DRAGONFLY\x13\x00\x00\x003.938184:84:PEACOCK\x12\x00\x00\x003.802613:16:BULBUL\x16\x00\x00\x003.765112:59:VINE SNAKE\x18\x00\x00\x003.555428:304:LEAF BEETLE\x12\x00\x00\x003.268481:91:COUCAL\x12\x00\x00\x003.105011:824:STOLE\x13\x00\x00\x003.083197:572:GOBLET\x12\x00\x00\x003.004841:633:LOUPE\x14\x00\x00\x003.003068:133:BITTERN\x19\x00\x00\x002.985944:82:RUFFED GROUSE\x14\x00\x00\x002.969416:946:CARDOON\x17\x00\x00\x002.962297:892:WALL CLOCK\x18\x00\x00\x002.949786:305:DUNG BEETLE\x11\x00\x00\x002.901230:42:AGAMA\x11\x00\x00\x002.861834:97:DRAKE\x11\x00\x00\x002.845032:714:PICK\x1d\x00\x00\x002.840478:364:THREE-TOED SLOTH\x19\x00\x00\x002.831974:948:GRANNY SMITH\x16\x00\x00\x002.830503:320:DAMSELFLY\x13\x00\x00\x002.769560:398:ABACUS\x18\x00\x00\x002.760760:861:TOILET SEAT\x1d\x00\x00\x002.592224:376:PROBOSCIS MONKEY\x14\x00\x00\x002.580803:13:SNOWBIRD\x17\x00\x00\x002.543412:644:MATCHSTICK\x13\x00\x00\x002.502671:307:WEEVIL\x14\x00\x00\x002.470719:816:SPINDLE\x18\x00\x00\x002.470528:852:TENNIS BALL\x11\x00\x00\x002.452069:600:HOOK\x1d\x00\x00\x002.441649:47:AFRICAN CHAMELEON\x12\x00\x00\x002.433671:749:QUILL\x15\x00\x00\x002.399983:679:NECKLACE\x1a\x00\x00\x002.394094:611:JIGSAW PUZZLE\x14\x00\x00\x002.383828:301:LADYBUG\x13\x00\x00\x002.377431:1:GOLDFISH!\x00\x00\x002.349733:140:RED-BACKED SANDPIPER\x17\x00\x00\x002.332492:55:GREEN SNAKE\x17\x00\x00\x002.323010:696:PAINTBRUSH\x10\x00\x00\x002.318759:309:BEE\x14\x00\x00\x002.283195:695:PADLOCK\x18\x00\x00\x002.279469:552:FEATHER BOA\x15\x00\x00\x002.217263:150:SEA LION\x11\x00\x00\x002.213060:883:VASE\x12\x00\x00\x002.150295:494:CHIME\x17\x00\x00\x002.139569:807:SOLAR DISH\x15\x00\x00\x002.139527:377:MARMOSET\x1a\x00\x00\x002.138633:559:FOLDING CHAIR\x19\x00\x00\x002.126787:898:WATER BOTTLE\x10\x00\x00\x002.096360:308:FLY\x13\x00\x00\x002.085981:897:WASHER\x12\x00\x00\x002.031478:988:ACORN\x14\x00\x00\x002.027971:89:COCKATOO\x12\x00\x00\x002.023943:951:LEMON\x18\x00\x00\x002.022991:128:BLACK STORK\x11\x00\x00\x002.019416:911:WOOL\x15\x00\x00\x001.944114:975:LAKESIDE\x17\x00\x00\x001.938696:912:WORM FENCE\x17\x00\x00\x001.922051:5:ELECTRIC RAY\x18\x00\x00\x001.902669:46:GREEN LIZARD\x12\x00\x00\x001.902419:488:CHAIN\x11\x00\x00\x001.896163:616:KNOT\x12\x00\x00\x001.883273:673:MOUSE\x16\x00\x00\x001.882831:448:BIRDHOUSE\x13\x00\x00\x001.879742:316:CICADA\x16\x00\x00\x001.859543:818:SPOTLIGHT\x13\x00\x00\x001.856104:950:ORANGE\x14\x00\x00\x001.851050:93:HORNBILL\x1d\x00\x00\x001.833514:527:DESKTOP COMPUTER\x15\x00\x00\x001.804665:30:BULL FROG\x15\x00\x00\x001.799966:617:LAB COAT\x1c\x00\x00\x001.794349:489:CHAINLINK FENCE\x17\x00\x00\x001.790604:584:HAIR SLIDE\x17\x00\x00\x001.790327:851:TELEVISION\x1a\x00\x00\x001.788963:302:GROUND BEETLE\x11\x00\x00\x001.788802:577:GONG\x12\x00\x00\x001.782072:80:GROUSE\x14\x00\x00\x001.771086:664:MONITOR\x1d\x00\x00\x001.765052:507:COMBINATION LOCK\x15\x00\x00\x001.752546:836:SUNGLASS\x11\x00\x00\x001.748576:769:RULE\x13\x00\x00\x001.737643:23:VULTURE\x1f\x00\x00\x001.706546:303:LONG-HORNED BEETLE\x14\x00\x00\x001.703732:417:BALLOON\x12\x00\x00\x001.696302:7:ROOSTER\x19\x00\x00\x001.680656:387:LESSER PANDA\x19\x00\x00\x001.658285:300:TIGER BEETLE\x11\x00\x00\x001.650747:733:POLE\x19\x00\x00\x001.629049:409:ANALOG CLOCK\x15\x00\x00\x001.609400:451:BOLO TIE\x14\x00\x00\x001.600475:725:PITCHER\x16\x00\x00\x001.591477:604:HOURGLASS\x1f\x00\x00\x001.585065:590:HAND-HELD COMPUTER\x1e\x00\x00\x001.580505:40:AMERICAN CHAMELEON\x19\x00\x00\x001.579520:522:CROQUET BALL\x15\x00\x00\x001.563156:326:LYCAENID\x16\x00\x00\x001.559538:826:STOPWATCH\x13\x00\x00\x001.538091:782:SCREEN\x15\x00\x00\x001.537094:557:FLAGPOLE\x15\x00\x00\x001.529117:916:WEB SITE\x13\x00\x00\x001.511950:641:MARACA\x13\x00\x00\x001.510321:775:SARONG\x10\x00\x00\x001.503608:952:FIG\x17\x00\x00\x001.498625:815:SPIDER WEB\x15\x00\x00\x001.487613:681:NOTEBOOK\x13\x00\x00\x001.485279:844:SWITCH\x12\x00\x00\x001.483408:778:SCALE\x17\x00\x00\x001.478171:822:STEEL DRUM\x18\x00\x00\x001.473927:728:PLASTIC BAG\x18\x00\x00\x001.471222:667:MORTARBOARD\x15\x00\x00\x001.469738:111:NEMATODE\x14\x00\x00\x001.461096:902:WHISTLE\x1a\x00\x00\x001.457809:739:POTTERS WHEEL\x15\x00\x00\x001.451698:404:AIRLINER\x10\x00\x00\x001.447749:78:TICK\x11\x00\x00\x001.422729:677:NAIL\x14\x00\x00\x001.421682:684:OCARINA\x18\x00\x00\x001.414797:921:BOOK JACKET\x16\x00\x00\x001.411799:619:LAMPSHADE\x15\x00\x00\x001.405326:103:PLATYPUS\x18\x00\x00\x001.393574:957:POMEGRANATE\x19\x00\x00\x001.391479:688:OSCILLOSCOPE\x17\x00\x00\x001.385885:583:GUILLOTINE\x17\x00\x00\x001.360924:515:COWBOY HAT\x11\x00\x00\x001.346396:331:HARE\x1b\x00\x00\x001.338950:648:MEDICINE CHEST\x17\x00\x00\x001.319630:64:GREEN MAMBA\x1c\x00\x00\x001.318985:139:RUDDY TURNSTONE\x14\x00\x00\x001.310631:277:RED FOX\x15\x00\x00\x001.304961:966:RED WINE\x13\x00\x00\x001.302737:735:PONCHO\x16\x00\x00\x001.300256:574:GOLF BALL\x17\x00\x00\x001.298837:949:STRAWBERRY\x18\x00\x00\x001.280737:659:MIXING BOWL\x16\x00\x00\x001.267075:426:BAROMETER\x17\x00\x00\x001.265100:846:TABLE LAMP\x17\x00\x00\x001.261311:317:LEAFHOPPER\x15\x00\x00\x001.256378:157:PAPILLON\x18\x00\x00\x001.249869:87:AFRICAN GREY\x19\x00\x00\x001.243727:213:IRISH SETTER\x19\x00\x00\x001.231984:335:FOX SQUIRREL\x16\x00\x00\x001.231199:418:BALLPOINT\x11\x00\x00\x001.225957:506:COIL\x11\x00\x00\x001.225885:758:REEL\x1f\x00\x00\x001.223339:487:CELLULAR TELEPHONE\x1d\x00\x00\x001.216252:710:PENCIL SHARPENER\x1c\x00\x00\x001.215075:382:SQUIRREL MONKEY\x14\x00\x00\x001.214476:742:PRINTER\x12\x00\x00\x001.210736:134:CRANE\x15\x00\x00\x001.207649:777:SCABBARD\x16\x00\x00\x001.202858:745:PROJECTOR\x13\x00\x00\x001.194747:446:BINDER\x1a\x00\x00\x001.192020:956:CUSTARD APPLE\x17\x00\x00\x001.190958:389:BARRACOUTA\x10\x00\x00\x001.183650:738:POT\x16\x00\x00\x001.177905:944:ARTICHOKE\x15\x00\x00\x001.176553:124:CRAYFISH\x1d\x00\x00\x001.168417:215:BRITTANY SPANIEL\x19\x00\x00\x001.156092:545:ELECTRIC FAN\x1a\x00\x00\x001.150013:24:GREAT GREY OWL\x14\x00\x00\x001.147067:645:MAYPOLE\x13\x00\x00\x001.146646:6:STINGRAY\x13\x00\x00\x001.133797:438:BEAKER\x1e\x00\x00\x001.123394:306:RHINOCEROS BEETLE\x13\x00\x00\x001.120431:374:LANGUR\x19\x00\x00\x001.111384:285:EGYPTIAN CAT\x14\x00\x00\x001.111153:855:THIMBLE\x11\x00\x00\x001.103784:526:DESK\x1b\x00\x00\x001.103336:794:SHOWER CURTAIN\x13\x00\x00\x001.100794:676:MUZZLE\x14\x00\x00\x001.097772:624:LIBRARY\x1a\x00\x00\x001.095995:999:TOILET TISSUE\x11\x00\x00\x001.093313:646:MAZE\x13\x00\x00\x001.086696:463:BUCKET\x14\x00\x00\x001.086604:36:TERRAPIN\x17\x00\x00\x001.070396:75:BLACK WIDOW\x1d\x00\x00\x001.064575:635:MAGNETIC COMPASS\x15\x00\x00\x001.061916:318:LACEWING\x10\x00\x00\x001.043019:989:HIP\x13\x00\x00\x001.042787:464:BUCKLE\x15\x00\x00\x001.036074:263:PEMBROKE\x13\x00\x00\x001.033136:849:TEAPOT\x15\x00\x00\x001.028483:708:PEDESTAL\x10\x00\x00\x001.020386:903:WIG\x16\x00\x00\x001.018911:2:WHITE SHARK\x14\x00\x00\x001.014240:845:SYRINGE\x15\x00\x00\x001.006815:978:SEASHORE\x11\x00\x00\x001.000576:380:TITI\x16\x00\x00\x000.990187:151:CHIHUAHUA\x16\x00\x00\x000.985383:655:MINISKIRT\x12\x00\x00\x000.980866:819:STAGE\x17\x00\x00\x000.975485:338:GUINEA PIG\x17\x00\x00\x000.969244:73:BARN SPIDER\x19\x00\x00\x000.969220:518:CRASH HELMET\x16\x00\x00\x000.967246:592:HARD DISC\x12\x00\x00\x000.966459:411:APRON\x17\x00\x00\x000.961197:585:HAIR SPRAY\x17\x00\x00\x000.959099:729:PLATE RACK\x1b\x00\x00\x000.958131:4:HAMMERHEAD SHARK\x1a\x00\x00\x000.957708:530:DIGITAL CLOCK\x10\x00\x00\x000.955555:443:BIB\x12\x00\x00\x000.951063:113:SNAIL\x14\x00\x00\x000.947665:699:PANPIPE\x10\x00\x00\x000.943872:998:EAR\x11\x00\x00\x000.942756:868:TRAY\x18\x00\x00\x000.934236:784:SCREWDRIVER\x14\x00\x00\x000.931772:731:PLUNGER\x12\x00\x00\x000.930846:558:FLUTE\x13\x00\x00\x000.928110:523:CRUTCH\x11\x00\x00\x000.918554:606:IRON\x12\x00\x00\x000.914103:273:DINGO\x19\x00\x00\x000.902983:905:WINDOW SHADE\x16\x00\x00\x000.892906:899:WATER JUG\x16\x00\x00\x000.886949:896:WASHBASIN\x17\x00\x00\x000.884800:223:SCHIPPERKE\x1b\x00\x00\x000.879419:132:AMERICAN EGRET\x18\x00\x00\x000.879370:691:OXYGEN MASK\x14\x00\x00\x000.878483:990:BUCKEYE\x16\x00\x00\x000.871345:924:GUACAMOLE\x12\x00\x00\x000.869678:783:SCREW\x14\x00\x00\x000.866942:626:LIGHTER\x1a\x00\x00\x000.862139:531:DIGITAL WATCH\x14\x00\x00\x000.849605:711:PERFUME\x12\x00\x00\x000.840607:519:CRATE\x14\x00\x00\x000.833847:253:BASENJI\x10\x00\x00\x000.827239:970:ALP\x19\x00\x00\x000.824831:145:KING PENGUIN\x15\x00\x00\x000.810054:264:CARDIGAN\x14\x00\x00\x000.808484:293:CHEETAH\x16\x00\x00\x000.797274:701:PARACHUTE\x17\x00\x00\x000.791518:447:BINOCULARS\x19\x00\x00\x000.786021:423:BARBER CHAIR\x15\x00\x00\x000.785714:984:RAPESEED\x12\x00\x00\x000.781514:501:CLOAK\x18\x00\x00\x000.770569:740:POWER DRILL\x1b\x00\x00\x000.768685:83:PRAIRIE CHICKEN\x18\x00\x00\x000.764903:392:ROCK BEAUTY\x11\x00\x00\x000.761860:771:SAFE\x17\x00\x00\x000.759481:973:CORAL REEF\x18\x00\x00\x000.758488:919:STREET SIGN\x13\x00\x00\x000.754104:743:PRISON\x17\x00\x00\x000.745510:772:SAFETY PIN\x15\x00\x00\x000.715185:419:BAND AID\x18\x00\x00\x000.713463:907:WINE BOTTLE\x13\x00\x00\x000.709896:227:KELPIE\x16\x00\x00\x000.700096:365:ORANGUTAN\x14\x00\x00\x000.695355:642:MARIMBA\x14\x00\x00\x000.693119:369:SIAMANG\x15\x00\x00\x000.686694:31:TREE FROG\x10\x00\x00\x000.678324:310:ANT\x17\x00\x00\x000.673808:580:GREENHOUSE\x15\x00\x00\x000.670567:481:CASSETTE\x13\x00\x00\x000.669797:513:CORNET\x1b\x00\x00\x000.669764:761:REMOTE CONTROL\x1f\x00\x00\x000.666130:117:CHAMBERED NAUTILUS\x11\x00\x00\x000.665734:800:SLOT\x15\x00\x00\x000.664958:875:TROMBONE\x15\x00\x00\x000.663633:378:CAPUCHIN\x1a\x00\x00\x000.662663:381:SPIDER MONKEY\x10\x00\x00\x000.655160:795:SKI\x1a\x00\x00\x000.654520:920:TRAFFIC LIGHT\x13\x00\x00\x000.650824:483:CASTLE\x12\x00\x00\x000.650741:764:RIFLE\x16\x00\x00\x000.649610:485:CD PLAYER\x13\x00\x00\x000.649159:954:BANANA\x13\x00\x00\x000.638993:787:SHIELD\x1a\x00\x00\x000.637152:767:RUBBER ERASER\x1a\x00\x00\x000.632876:313:WALKING STICK\x16\x00\x00\x000.623600:22:BALD EAGLE\x10\x00\x00\x000.614123:456:BOW\x18\x00\x00\x000.610197:805:SOCCER BALL\x13\x00\x00\x000.610089:272:COYOTE\x17\x00\x00\x000.602960:737:POP BOTTLE\x16\x00\x00\x000.598547:810:SPACE BAR\x13\x00\x00\x000.585712:337:BEAVER\x15\x00\x00\x000.585057:895:WARPLANE\x11\x00\x00\x000.582699:643:MASK\x13\x00\x00\x000.581812:620:LAPTOP\x13\x00\x00\x000.579559:410:APIARY\x19\x00\x00\x000.579444:591:HANDKERCHIEF\x15\x00\x00\x000.575735:549:ENVELOPE\x15\x00\x00\x000.574417:454:BOOKSHOP\x1a\x00\x00\x000.573819:904:WINDOW SCREEN\x17\x00\x00\x000.572764:32:TAILED FROG\x15\x00\x00\x000.571510:796:SKI MASK\x13\x00\x00\x000.570359:693:PADDLE\x10\x00\x00\x000.568468:968:CUP\x1a\x00\x00\x000.562986:379:HOWLER MONKEY\x17\x00\x00\x000.560286:841:SWEATSHIRT\x16\x00\x00\x000.557637:593:HARMONICA\x12\x00\x00\x000.552896:274:DHOLE\x12\x00\x00\x000.537236:462:BROOM\x13\x00\x00\x000.536248:658:MITTEN\x17\x00\x00\x000.531914:424:BARBERSHOP\x17\x00\x00\x000.525322:917:COMIC BOOK\x17\x00\x00\x000.523453:719:PIGGY BANK\x16\x00\x00\x000.518573:107:JELLYFISH\x13\x00\x00\x000.518030:452:BONNET\x14\x00\x00\x000.515065:469:CALDRON\x15\x00\x00\x000.509278:562:FOUNTAIN\x14\x00\x00\x000.469466:457:BOW TIE\x14\x00\x00\x000.465020:859:TOASTER\x11\x00\x00\x000.464947:553:FILE\x1a\x00\x00\x000.455411:670:MOTOR SCOOTER\x15\x00\x00\x000.455132:808:SOMBRERO\x15\x00\x00\x000.441509:81:PTARMIGAN\x13\x00\x00\x000.440781:428:BARROW\x18\x00\x00\x000.437571:311:GRASSHOPPER\x13\x00\x00\x000.435628:478:CARTON\x13\x00\x00\x000.429484:588:HAMPER\x1d\x00\x00\x000.429437:918:CROSSWORD PUZZLE\x1a\x00\x00\x000.429139:43:FRILLED LIZARD\x11\x00\x00\x000.426559:987:CORN\x12\x00\x00\x000.424916:360:OTTER\x13\x00\x00\x000.418949:792:SHOVEL\x16\x00\x00\x000.418415:542:DRUMSTICK\x18\x00\x00\x000.417937:700:PAPER TOWEL\x1e\x00\x00\x000.417327:262:BRABANCON GRIFFON\x15\x00\x00\x000.414118:880:UNICYCLE\x12\x00\x00\x000.406654:281:TABBY\x11\x00\x00\x000.399845:683:OBOE\x12\x00\x00\x000.395658:472:CANOE\x19\x00\x00\x000.394295:72:GARDEN SPIDER\x11\x00\x00\x000.392198:114:SLUG\x14\x00\x00\x000.362575:977:SANDBAR\x15\x00\x00\x000.345072:937:BROCCOLI\x17\x00\x00\x000.335746:712:PETRI DISH\x13\x00\x00\x000.331638:587:HAMMER\x10\x00\x00\x000.331537:876:TUB\x15\x00\x00\x000.325964:280:GREY FOX\x12\x00\x00\x000.320312:832:STUPA\x14\x00\x00\x000.319964:323:MONARCH\x15\x00\x00\x000.319727:141:REDSHANK\x16\x00\x00\x000.318301:674:MOUSETRAP\x17\x00\x00\x000.310368:678:NECK BRACE\x16\x00\x00\x000.302611:407:AMBULANCE\x10\x00\x00\x000.301713:254:PUG\x18\x00\x00\x000.295728:158:TOY TERRIER\x1c\x00\x00\x000.293643:202:WHEATEN TERRIER\x13\x00\x00\x000.292937:724:PIRATE\x19\x00\x00\x000.292053:716:PICKET FENCE\x14\x00\x00\x000.286860:801:SNORKEL \x00\x00\x000.285877:275:AFRICAN HUNTING DOG\x12\x00\x00\x000.277018:112:CONCH\x11\x00\x00\x000.272960:840:SWAB\x1c\x00\x00\x000.272797:546:ELECTRIC GUITAR\x11\x00\x00\x000.271210:834:SUIT\x18\x00\x00\x000.267745:632:LOUDSPEAKER\x12\x00\x00\x000.262307:862:TORCH\x1a\x00\x00\x000.261434:704:PARKING METER\x1a\x00\x00\x000.255376:137:AMERICAN COOT\x17\x00\x00\x000.248967:837:SUNGLASSES\x14\x00\x00\x000.242655:835:SUNDIAL\x19\x00\x00\x000.241748:74:GARDEN SPIDER\x13\x00\x00\x000.235600:692:PACKET\x18\x00\x00\x000.234962:727:PLANETARIUM\x15\x00\x00\x000.232402:429:BASEBALL\x14\x00\x00\x000.224198:299:MEERKAT"\x00\x00\x000.221121:444:BICYCLE-BUILT-FOR-TWO\x11\x00\x00\x000.218000:541:DRUM\x13\x00\x00\x000.217489:721:PILLOW%\x00\x00\x000.214787:209:CHESAPEAKE BAY RETRIEVER\x12\x00\x00\x000.211322:923:PLATE\x1f\x00\x00\x000.210130:183:KERRY BLUE TERRIER\x10\x00\x00\x000.209658:0:TENCH\x16\x00\x00\x000.208301:142:DOWITCHER\x18\x00\x00\x000.207926:241:ENTLEBUCHER\x10\x00\x00\x000.206087:395:GAR\'\x00\x00\x000.203508:238:GREATER SWISS MOUNTAIN DOG\x1e\x00\x00\x000.203040:508:COMPUTER KEYBOARD\x18\x00\x00\x000.202791:566:FRENCH HORN\x18\x00\x00\x000.199785:945:BELL PEPPER\x1a\x00\x00\x000.195379:623:LETTER OPENER\x17\x00\x00\x000.190138:473:CAN OPENER\x15\x00\x00\x000.188708:780:SCHOONER\x1e\x00\x00\x000.187295:324:CABBAGE BUTTERFLY\x13\x00\x00\x000.185544:885:VELVET\x1e\x00\x00\x000.184747:325:SULPHUR BUTTERFLY\x15\x00\x00\x000.183127:453:BOOKCASE\x14\x00\x00\x000.182686:682:OBELISK\x14\x00\x00\x000.177639:312:CRICKET\x15\x00\x00\x000.174900:879:UMBRELLA\x1f\x00\x00\x000.169730:198:STANDARD SCHNAUZER\x18\x00\x00\x000.169292:109:BRAIN CORAL\x13\x00\x00\x000.167451:427:BARREL\x17\x00\x00\x000.167388:58:WATER SNAKE\x1e\x00\x00\x000.167121:34:LEATHERBACK TURTLE\x13\x00\x00\x000.166422:889:VIOLIN\x17\x00\x00\x000.161970:60:NIGHT SNAKE\x12\x00\x00\x000.158377:850:TEDDY\x17\x00\x00\x000.154478:265:TOY POODLE\x18\x00\x00\x000.152559:823:STETHOSCOPE\x1b\x00\x00\x000.151060:804:SOAP DISPENSER\x1a\x00\x00\x000.150780:400:ACADEMIC GOWN\x19\x00\x00\x000.149005:173:IBIZAN HOUND\x16\x00\x00\x000.148204:599:HONEYCOMB\x17\x00\x00\x000.146233:367:CHIMPANZEE\x1f\x00\x00\x000.143248:237:MINIATURE PINSCHER\x1b\x00\x00\x000.137232:199:SCOTCH TERRIER\x11\x00\x00\x000.124475:908:WING\x14\x00\x00\x000.118226:135:LIMPKIN\x19\x00\x00\x000.096783:986:LADY SLIPPER\x1e\x00\x00\x000.093566:171:ITALIAN GREYHOUND\x14\x00\x00\x000.081715:321:ADMIRAL\x14\x00\x00\x000.081552:373:MACAQUE\x17\x00\x00\x000.076640:734:POLICE VAN\x12\x00\x00\x000.071396:517:CRANE\x18\x00\x00\x000.068002:440:BEER BOTTLE\x1f\x00\x00\x000.064233:193:AUSTRALIAN TERRIER\x13\x00\x00\x000.062872:979:VALLEY\x14\x00\x00\x000.059840:813:SPATULA\x17\x00\x00\x000.058556:504:COFFEE MUG\x18\x00\x00\x000.051553:330:WOOD RABBIT\x18\x00\x00\x000.048906:284:SIAMESE CAT\x17\x00\x00\x000.045533:703:PARK BENCH\x15\x00\x00\x000.041801:828:STRAINER\x11\x00\x00\x000.031766:502:CLOG\x14\x00\x00\x000.029974:138:BUSTARD\x16\x00\x00\x000.027434:459:BRASSIERE\x15\x00\x00\x000.025855:271:RED WOLF\x16\x00\x00\x000.021328:512:CORKSCREW\x13\x00\x00\x000.021085:872:TRIPOD\x10\x00\x00\x000.020582:468:CAB\x18\x00\x00\x000.019685:195:BOSTON BULL\x19\x00\x00\x000.016529:669:MOSQUITO NET\x13\x00\x00\x000.007288:372:BABOON\x17\x00\x00\x000.006412:163:BLOODHOUND\x18\x00\x00\x000.004492:756:RAIN BARREL\x0f\x00\x00\x00-0.001102:8:HEN\x16\x00\x00\x00-0.001655:225:MALINOIS\x15\x00\x00\x00-0.010844:333:HAMSTER\x1f\x00\x00\x00-0.010940:187:YORKSHIRE TERRIER\x16\x00\x00\x00-0.011804:568:FUR COAT\x1b\x00\x00\x00-0.013627:672:MOUNTAIN TENT\x19\x00\x00\x00-0.015886:461:BREASTPLATE\x15\x00\x00\x00-0.020457:639:MAILLOT\x1b\x00\x00\x00-0.024114:791:SHOPPING CART\x17\x00\x00\x00-0.035934:449:BOATHOUSE\x14\x00\x00\x00-0.038152:752:RACKET\x15\x00\x00\x00-0.042968:71:SCORPION\x16\x00\x00\x00-0.043376:622:LENS CAP\x15\x00\x00\x00-0.044080:654:MINIBUS\x19\x00\x00\x00-0.045024:713:PHOTOCOPIER\x15\x00\x00\x00-0.052158:288:LEOPARD\x18\x00\x00\x00-0.061615:798:SLIDE RULE\x14\x00\x00\x00-0.062778:412:ASHCAN\x19\x00\x00\x00-0.073587:938:CAULIFLOWER\x1a\x00\x00\x00-0.075748:416:BALANCE BEAM\x17\x00\x00\x00-0.077675:282:TIGER CAT\x15\x00\x00\x00-0.082030:637:MAILBOX\x18\x00\x00\x00-0.086738:781:SCOREBOARD\x1d\x00\x00\x00-0.086918:402:ACOUSTIC GUITAR\x1b\x00\x00\x00-0.090360:509:CONFECTIONERY\x1e\x00\x00\x00-0.091755:167:ENGLISH FOXHOUND\x1a\x00\x00\x00-0.093549:799:SLIDING DOOR\x1b\x00\x00\x00-0.098073:582:GROCERY STORE\x19\x00\x00\x00-0.098156:514:COWBOY BOOT\x17\x00\x00\x00-0.099712:484:CATAMARAN\x1b\x00\x00\x00-0.100425:495:CHINA CABINET\x1a\x00\x00\x00-0.103606:243:BULL MASTIFF\x14\x00\x00\x00-0.129238:445:BIKINI\x19\x00\x00\x00-0.130416:127:WHITE STORK\x18\x00\x00\x00-0.131940:736:POOL TABLE\x16\x00\x00\x00-0.133336:164:BLUETICK\x1d\x00\x00\x00-0.140920:503:COCKTAIL SHAKER\x1d\x00\x00\x00-0.149637:732:POLAROID CAMERA\x15\x00\x00\x00-0.152527:405:AIRSHIP\x14\x00\x00\x00-0.154807:315:MANTIS\x13\x00\x00\x00-0.155674:726:PLANE\x14\x00\x00\x00-0.159038:362:BADGER\x1b\x00\x00\x00-0.161313:184:IRISH TERRIER\x17\x00\x00\x00-0.163017:33:LOGGERHEAD\x14\x00\x00\x00-0.163814:370:GUENON\x15\x00\x00\x00-0.166052:932:PRETZEL\x19\x00\x00\x00-0.171461:660:MOBILE HOME\x17\x00\x00\x00-0.174392:929:ICE LOLLY\x15\x00\x00\x00-0.174609:656:MINIVAN\x15\x00\x00\x00-0.176251:596:HATCHET\x14\x00\x00\x00-0.177184:680:NIPPLE\x17\x00\x00\x00-0.178278:809:SOUP BOWL\x13\x00\x00\x00-0.180503:486:CELLO\x19\x00\x00\x00-0.190768:240:APPENZELLER\x1b\x00\x00\x00-0.191769:214:GORDON SETTER"\x00\x00\x00-0.192553:233:BOUVIER DES FLANDRES\x1a\x00\x00\x00-0.192652:467:BUTCHER SHOP\x13\x00\x00\x00-0.194146:843:SWING\x16\x00\x00\x00-0.194607:753:RADIATOR\x15\x00\x00\x00-0.203870:168:REDBONE\x17\x00\x00\x00-0.208204:814:SPEEDBOAT\x11\x00\x00\x00-0.220296:390:EEL\x17\x00\x00\x00-0.222319:829:STREETCAR\x11\x00\x00\x00-0.226297:909:WOK\x12\x00\x00\x00-0.229762:806:SOCK\x1a\x00\x00\x00-0.242444:166:WALKER HOUND\x1b\x00\x00\x00-0.244623:671:MOUNTAIN BIKE\x14\x00\x00\x00-0.247596:698:PALACE\x14\x00\x00\x00-0.247835:231:COLLIE\x1c\x00\x00\x00-0.260893:786:SEWING MACHINE\x13\x00\x00\x00-0.267136:361:SKUNK\x19\x00\x00\x00-0.274342:720:PILL BOTTLE\x1a\x00\x00\x00-0.274678:860:TOBACCO SHOP\x11\x00\x00\x00-0.274781:26:NEWT\x14\x00\x00\x00-0.275571:581:GRILLE\x15\x00\x00\x00-0.276468:41:WHIPTAIL\x18\x00\x00\x00-0.278707:686:OIL FILTER\x16\x00\x00\x00-0.283840:925:CONSOMME\x15\x00\x00\x00-0.287714:638:MAILLOT\x16\x00\x00\x00-0.289612:65:SEA SNAKE\x14\x00\x00\x00-0.289836:126:ISOPOD\x19\x00\x00\x00-0.301672:891:WAFFLE IRON\x18\x00\x00\x00-0.303164:460:BREAKWATER\x17\x00\x00\x00-0.323472:421:BANNISTER\x14\x00\x00\x00-0.326360:697:PAJAMA\x18\x00\x00\x00-0.328036:779:SCHOOL BUS\x13\x00\x00\x00-0.328495:750:QUILT\x15\x00\x00\x00-0.330024:636:MAILBAG\x15\x00\x00\x00-0.332442:865:TOYSHOP\x11\x00\x00\x00-0.333760:776:SAX\x1a\x00\x00\x00-0.334358:120:FIDDLER CRAB\x16\x00\x00\x00-0.336231:763:REVOLVER\x12\x00\x00\x00-0.342122:260:CHOW\x12\x00\x00\x00-0.344876:536:DOCK\x11\x00\x00\x00-0.351428:958:HAY\x17\x00\x00\x00-0.353626:401:ACCORDION\x17\x00\x00\x00-0.354341:838:SUNSCREEN\x1a\x00\x00\x00-0.364392:910:WOODEN SPOON\x1d\x00\x00\x00-0.365610:267:STANDARD POODLE\x13\x00\x00\x00-0.366071:827:STOVE\x15\x00\x00\x00-0.377133:172:WHIPPET\x19\x00\x00\x00-0.377818:269:TIMBER WOLF\x13\x00\x00\x00-0.379658:754:RADIO\x13\x00\x00\x00-0.381107:276:HYENA\x17\x00\x00\x00-0.381651:505:COFFEEPOT\x18\x00\x00\x00-0.384606:650:MICROPHONE\x1e\x00\x00\x00-0.388018:152:JAPANESE SPANIEL \x00\x00\x00-0.388777:177:SCOTTISH DEERHOUND\x1c\x00\x00\x00-0.389080:477:CARPENTERS KIT\x14\x00\x00\x00-0.389720:397:PUFFER\x19\x00\x00\x00-0.395841:983:SCUBA DIVER\x15\x00\x00\x00-0.399373:358:POLECAT\x15\x00\x00\x00-0.401944:539:DOORMAT\x18\x00\x00\x00-0.402541:493:CHIFFONIER!\x00\x00\x00-0.406008:159:RHODESIAN RIDGEBACK\x12\x00\x00\x00-0.409592:425:BARN\x16\x00\x00\x00-0.410564:613:JOYSTICK\x1d\x00\x00\x00-0.411586:886:VENDING MACHINE \x00\x00\x00-0.420715:221:IRISH WATERSPANIEL\x1a\x00\x00\x00-0.428175:598:HOME THEATER\x13\x00\x00\x00-0.430456:618:LADLE\x17\x00\x00\x00-0.430915:858:TILE ROOF\x13\x00\x00\x00-0.431895:458:BRASS\x13\x00\x00\x00-0.434842:371:PATAS\x1d\x00\x00\x00-0.436342:854:THEATER CURTAIN\x15\x00\x00\x00-0.436770:432:BASSOON\x1e\x00\x00\x00-0.444500:49:AFRICAN CROCODILE\x1a\x00\x00\x00-0.445040:532:DINING TABLE\x17\x00\x00\x00-0.449359:830:STRETCHER\x18\x00\x00\x00-0.450416:279:ARCTIC FOX\x1e\x00\x00\x00-0.450429:156:BLENHEIM SPANIEL\x1c\x00\x00\x00-0.453814:602:HORIZONTAL BAR\x14\x00\x00\x00-0.455160:668:MOSQUE\x14\x00\x00\x00-0.457056:497:CHURCH\x15\x00\x00\x00-0.459901:576:GONDOLA\x14\x00\x00\x00-0.463431:853:THATCH\x1d\x00\x00\x00-0.470255:790:SHOPPING BASKET\x15\x00\x00\x00-0.471372:278:KIT FOX\x19\x00\x00\x00-0.474413:901:WHISKEY JUG\x18\x00\x00\x00-0.474955:475:CAR MIRROR\x12\x00\x00\x00-0.477929:391:COHO\x14\x00\x00\x00-0.478245:290:JAGUAR\x1a\x00\x00\x00-0.484680:480:CASH MACHINE$\x00\x00\x00-0.490811:206:CURLY-COATED RETRIEVER$\x00\x00\x00-0.492644:218:WELSH SPRINGER SPANIEL\x1b\x00\x00\x00-0.494406:413:ASSAULT RIFLE\x17\x00\x00\x00-0.495218:251:DALMATIAN\x18\x00\x00\x00-0.495672:294:BROWN BEAR\x12\x00\x00\x00-0.504499:922:MENU\x12\x00\x00\x00-0.505565:578:GOWN\x1e\x00\x00\x00-0.510900:266:MINIATURE POODLE\x10\x00\x00\x00-0.511877:345:OX\x16\x00\x00\x00-0.518695:130:FLAMINGO\x1a\x00\x00\x00-0.522411:563:FOUNTAIN PEN\x14\x00\x00\x00-0.529039:226:BRIARD\x1b\x00\x00\x00-0.532585:53:RINGNECK SNAKE\x16\x00\x00\x00-0.533349:76:TARANTULA\x1d\x00\x00\x00-0.534765:200:TIBETAN TERRIER\x17\x00\x00\x00-0.536063:129:SPOONBILL\x16\x00\x00\x00-0.537885:236:DOBERMAN\x16\x00\x00\x00-0.538742:943:CUCUMBER\x18\x00\x00\x00-0.541501:490:CHAIN MAIL\x14\x00\x00\x00-0.541548:631:LOTION\x1c\x00\x00\x00-0.541982:182:BORDER TERRIER\x14\x00\x00\x00-0.542918:498:CINEMA\x13\x00\x00\x00-0.544518:662:MODEM\x16\x00\x00\x00-0.548896:191:AIREDALE\x17\x00\x00\x00-0.552844:785:SEAT BELT\x15\x00\x00\x00-0.556238:888:VIADUCT\x1c\x00\x00\x00-0.557857:722:PING-PONG BALL\x14\x00\x00\x00-0.558250:368:GIBBON\x1e\x00\x00\x00-0.560383:189:LAKELAND TERRIER\x12\x00\x00\x00-0.560617:357:MINK \x00\x00\x00-0.562531:174:NORWEGIAN ELKHOUND\x1a\x00\x00\x00-0.562703:831:STUDIO COUCH!\x00\x00\x00-0.568506:295:AMERICAN BLACK BEAR\x1c\x00\x00\x00-0.572860:220:SUSSEX SPANIEL!\x00\x00\x00-0.572879:878:TYPEWRITER KEYBOARD\x14\x00\x00\x00-0.575665:336:MARMOT\x17\x00\x00\x00-0.576074:521:CROCK POT\x16\x00\x00\x00-0.576500:298:MONGOOSE\x13\x00\x00\x00-0.577472:985:DAISY\x18\x00\x00\x00-0.577558:762:RESTAURANT\x18\x00\x00\x00-0.580354:709:PENCIL BOX\x13\x00\x00\x00-0.586378:492:CHEST\x15\x00\x00\x00-0.588333:435:BATHTUB\x1b\x00\x00\x00-0.591891:705:PASSENGER CAR\x13\x00\x00\x00-0.592356:628:LINER \x00\x00\x00-0.594996:208:LABRADOR RETRIEVER\x1c\x00\x00\x00-0.606174:383:MADAGASCAR CAT\x19\x00\x00\x00-0.614128:579:GRAND PIANO\x19\x00\x00\x00-0.617098:869:TRENCH COAT\x17\x00\x00\x00-0.621552:119:ROCK CRAB\x17\x00\x00\x00-0.630173:455:BOTTLECAP\x19\x00\x00\x00-0.631579:125:HERMIT CRAB\x18\x00\x00\x00-0.632271:741:PRAYER RUG\x19\x00\x00\x00-0.633287:589:HAND BLOWER\x19\x00\x00\x00-0.633837:388:GIANT PANDA\x13\x00\x00\x00-0.639783:972:CLIFF\x19\x00\x00\x00-0.647117:66:HORNED VIPER\x16\x00\x00\x00-0.648923:474:CARDIGAN!\x00\x00\x00-0.651210:359:BLACK-FOOTED FERRET\x1b\x00\x00\x00-0.656218:765:ROCKING CHAIR\x19\x00\x00\x00-0.657437:715:PICKELHAUBE\x19\x00\x00\x00-0.658725:848:TAPE PLAYER\x1e\x00\x00\x00-0.662562:940:SPAGHETTI SQUASH\x13\x00\x00\x00-0.663827:105:KOALA\x19\x00\x00\x00-0.664947:38:BANDED GECKO\x18\x00\x00\x00-0.665088:434:BATH TOWEL\x12\x00\x00\x00-0.666371:605:IPOD\x16\x00\x00\x00-0.666955:396:LIONFISH\x1d\x00\x00\x00-0.669285:235:GERMAN SHEPHERD\x19\x00\x00\x00-0.672666:436:BEACH WAGON\x17\x00\x00\x00-0.675265:3:TIGER SHARK\x15\x00\x00\x00-0.676331:422:BARBELL\x11\x00\x00\x00-0.676690:348:RAM\x17\x00\x00\x00-0.677893:689:OVERSKIRT\x14\x00\x00\x00-0.681761:690:OXCART\x19\x00\x00\x00-0.687405:906:WINDSOR TIE\x17\x00\x00\x00-0.688651:928:ICE CREAM\x1a\x00\x00\x00-0.696495:466:BULLET TRAIN\x15\x00\x00\x00-0.698900:366:GORILLA\x18\x00\x00\x00-0.701230:802:SNOWMOBILE\x17\x00\x00\x00-0.701542:955:JACKFRUIT\x14\x00\x00\x00-0.706222:116:CHITON\x17\x00\x00\x00-0.706730:314:COCKROACH\'\x00\x00\x00-0.708503:179:STAFFORDSHIRE BULLTERRIER\x18\x00\x00\x00-0.711078:248:ESKIMO DOG\x10\x00\x00\x00-0.712090:61:BOA\x15\x00\x00\x00-0.714779:524:CUIRASS\x12\x00\x00\x00-0.717131:594:HARP\x16\x00\x00\x00-0.718055:894:WARDROBE\x1f\x00\x00\x00-0.718805:839:SUSPENSION BRIDGE\x15\x00\x00\x00-0.721201:144:PELICAN\x14\x00\x00\x00-0.723153:964:POTPIE\x14\x00\x00\x00-0.732436:356:WEASEL\x1d\x00\x00\x00-0.733168:960:CHOCOLATE SAUCE\x1b\x00\x00\x00-0.746696:759:REFLEX CAMERA\x13\x00\x00\x00-0.749408:931:BAGEL\x14\x00\x00\x00-0.752722:470:CANDLE\x16\x00\x00\x00-0.754596:327:STARFISH\x15\x00\x00\x00-0.755695:375:COLOBUS\x12\x00\x00\x00-0.762742:538:DOME\x10\x00\x00\x00-0.765644:27:EFT\x13\x00\x00\x00-0.765882:211:VISLA\x17\x00\x00\x00-0.771635:37:BOX TURTLE\x18\x00\x00\x00-0.773128:825:STONE WALL\x1e\x00\x00\x00-0.795405:207:GOLDEN RETRIEVER\x12\x00\x00\x00-0.796663:609:JEEP\x16\x00\x00\x00-0.798556:967:ESPRESSO\x12\x00\x00\x00-0.802936:63:COBRA\x1e\x00\x00\x00-0.806272:652:MILITARY UNIFORM\x1b\x00\x00\x00-0.808793:702:PARALLEL BARS\x1d\x00\x00\x00-0.808816:186:NORWICH TERRIER\x18\x00\x00\x00-0.821995:890:VOLLEYBALL\x1b\x00\x00\x00-0.826841:569:GARBAGE TRUCK\x17\x00\x00\x00-0.826977:877:TURNSTILE\x14\x00\x00\x00-0.828055:610:JERSEY\x17\x00\x00\x00-0.832333:442:BELL COTE\x18\x00\x00\x00-0.841974:259:POMERANIAN\x18\x00\x00\x00-0.857614:863:TOTEM POLE\x1a\x00\x00\x00-0.865451:760:REFRIGERATOR\x14\x00\x00\x00-0.869967:614:KIMONO\x1f\x00\x00\x00-0.874209:50:AMERICAN ALLIGATOR\x15\x00\x00\x00-0.880760:537:DOGSLED \x00\x00\x00-0.883104:496:CHRISTMAS STOCKING\x1d\x00\x00\x00-0.888835:44:ALLIGATOR LIZARD\x13\x00\x00\x00-0.890930:420:BANJO\x1b\x00\x00\x00-0.892562:647:MEASURING CUP\x1f\x00\x00\x00-0.892621:230:SHETLAND SHEEPDOG\x16\x00\x00\x00-0.895032:394:STURGEON\x12\x00\x00\x00-0.899341:608:JEAN\x14\x00\x00\x00-0.911108:415:BAKERY\x1e\x00\x00\x00-0.914436:268:MEXICAN HAIRLESS\x1b\x00\x00\x00-0.916878:247:SAINT BERNARD\x1d\x00\x00\x00-0.918658:170:IRISH WOLFHOUND\x13\x00\x00\x00-0.920567:399:ABAYA\x1c\x00\x00\x00-0.921363:25:FIRE SALAMANDER\x18\x00\x00\x00-0.922810:976:PROMONTORY\x19\x00\x00\x00-0.923664:555:FIRE ENGINE\x19\x00\x00\x00-0.925898:556:FIRE SCREEN\x1c\x00\x00\x00-0.928072:510:CONTAINER SHIP\x1b\x00\x00\x00-0.928166:346:WATER BUFFALO\x13\x00\x00\x00-0.933251:665:MOPED\x18\x00\x00\x00-0.939540:297:SLOTH BEAR\x1a\x00\x00\x00-0.945308:941:ACORN SQUASH\x14\x00\x00\x00-0.949032:630:LOAFER\x12\x00\x00\x00-0.956520:730:PLOW\x1a\x00\x00\x00-0.958081:48:KOMODO DRAGON\x17\x00\x00\x00-0.961303:707:PAY-PHONE\x16\x00\x00\x00-0.967634:98:MERGANSER\x1b\x00\x00\x00-0.971969:201:SILKY TERRIER\x18\x00\x00\x00-0.974399:62:ROCK PYTHON\x13\x00\x00\x00-0.976014:292:TIGER\x1c\x00\x00\x00-0.976725:212:ENGLISH SETTER\x14\x00\x00\x00-0.977094:516:CRADLE\x14\x00\x00\x00-0.979199:774:SANDAL\x17\x00\x00\x00-0.983934:165:COONHOUND\x17\x00\x00\x00-0.986968:68:SIDEWINDER\x19\x00\x00\x00-0.987544:224:GROENENDAEL\x13\x00\x00\x00-0.988111:789:SHOJI\x12\x00\x00\x00-0.992957:287:LYNX"\x00\x00\x00-0.995737:548:ENTERTAINMENT CENTER\x18\x00\x00\x00-0.996203:328:SEA URCHIN#\x00\x00\x00-0.998954:180:STAFFORDSHIRE TERRIER\x18\x00\x00\x00-1.002741:234:ROTTWEILER\x18\x00\x00\x00-1.003282:534:DISHWASHER\x19\x00\x00\x00-1.005098:57:GARTER SNAKE\x18\x00\x00\x00-1.006310:270:WHITE WOLF\x15\x00\x00\x00-1.007170:322:RINGLET\x1c\x00\x00\x00-1.012581:245:FRENCH BULLDOG\x1a\x00\x00\x00-1.016641:811:SPACE HEATER\x14\x00\x00\x00-1.032445:162:BEAGLE\x1b\x00\x00\x00-1.033585:203:WHITE TERRIER\x15\x00\x00\x00-1.034417:353:GAZELLE\x14\x00\x00\x00-1.038186:332:ANGORA\x18\x00\x00\x00-1.040749:178:WEIMARANER\x14\x00\x00\x00-1.046285:969:EGGNOG\x18\x00\x00\x00-1.051258:246:GREAT DANE\x1c\x00\x00\x00-1.053092:250:SIBERIAN HUSKY\x16\x00\x00\x00-1.053640:79:CENTIPEDE\x13\x00\x00\x00-1.056608:355:LLAMA"\x00\x00\x00-1.062780:239:BERNESE MOUNTAIN DOG\x18\x00\x00\x00-1.063392:773:SALTSHAKER\x12\x00\x00\x00-1.071988:520:CRIB\x14\x00\x00\x00-1.088310:352:IMPALA\x15\x00\x00\x00-1.094222:499:CLEAVER\x19\x00\x00\x00-1.094240:108:SEA ANEMONE\x16\x00\x00\x00-1.100526:155:SHIH-TZU\x1b\x00\x00\x00-1.103373:935:MASHED POTATO\x15\x00\x00\x00-1.113244:349:BIGHORN!\x00\x00\x00-1.122853:547:ELECTRIC LOCOMOTIVE\x17\x00\x00\x00-1.129224:953:PINEAPPLE\x19\x00\x00\x00-1.130450:283:PERSIAN CAT\x18\x00\x00\x00-1.135284:567:FRYING PAN\x17\x00\x00\x00-1.138611:35:MUD TURTLE\x14\x00\x00\x00-1.140236:149:DUGONG\x18\x00\x00\x00-1.140280:768:RUGBY BALL\x13\x00\x00\x00-1.141040:192:CAIRN\x14\x00\x00\x00-1.142420:339:SORREL\x16\x00\x00\x00-1.153147:653:MILK CAN"\x00\x00\x00-1.164613:210:SHORT-HAIRED POINTER\x1b\x00\x00\x00-1.173273:252:AFFENPINSCHER\x19\x00\x00\x00-1.174704:433:BATHING CAP\x1a\x00\x00\x00-1.184711:148:KILLER WHALE\x16\x00\x00\x00-1.192534:870:TRICYCLE\x13\x00\x00\x00-1.196150:687:ORGAN\x14\x00\x00\x00-1.197402:666:MORTAR!\x00\x00\x00-1.198079:196:MINIATURE SCHNAUZER\x17\x00\x00\x00-1.202449:651:MICROWAVE\x17\x00\x00\x00-1.207458:959:CARBONARA\x15\x00\x00\x00-1.219228:926:HOT POT\x17\x00\x00\x00-1.219442:663:MONASTERY\x16\x00\x00\x00-1.219496:947:MUSHROOM\x15\x00\x00\x00-1.220656:102:ECHIDNA\x16\x00\x00\x00-1.227589:69:TRILOBITE\x1a\x00\x00\x00-1.228394:289:SNOW LEOPARD\x1c\x00\x00\x00-1.233003:219:COCKER SPANIEL\x17\x00\x00\x00-1.235588:334:PORCUPINE\x13\x00\x00\x00-1.246506:384:INDRI\x15\x00\x00\x00-1.246928:216:CLUMBER\x18\x00\x00\x00-1.248940:621:LAWN MOWER\x13\x00\x00\x00-1.250085:913:WRECK\x16\x00\x00\x00-1.252065:871:TRIMARAN\x13\x00\x00\x00-1.256884:748:PURSE\x11\x00\x00\x00-1.259643:341:HOG\x18\x00\x00\x00-1.262444:77:WOLF SPIDER\x1a\x00\x00\x00-1.266560:747:PUNCHING BAG\x17\x00\x00\x00-1.290475:146:ALBATROSS\x14\x00\x00\x00-1.291235:286:COUGAR\x1b\x00\x00\x00-1.296954:232:BORDER COLLIE\x1d\x00\x00\x00-1.299536:197:GIANT SCHNAUZER\x18\x00\x00\x00-1.300732:612:JINRIKISHA\x18\x00\x00\x00-1.301130:603:HORSE CART\x14\x00\x00\x00-1.306039:893:WALLET\x14\x00\x00\x00-1.310720:997:BOLETE\x13\x00\x00\x00-1.318508:9:OSTRICH\x15\x00\x00\x00-1.318764:570:GASMASK\x1d\x00\x00\x00-1.319363:185:NORFOLK TERRIER\x1c\x00\x00\x00-1.326616:257:GREAT PYRENEES\x16\x00\x00\x00-1.332833:115:SEA SLUG\x13\x00\x00\x00-1.342358:242:BOXER\x1a\x00\x00\x00-1.353705:393:ANEMONE FISH\x15\x00\x00\x00-1.353994:980:VOLCANO\x1e\x00\x00\x00-1.356510:386:AFRICAN ELEPHANT\x17\x00\x00\x00-1.356894:408:AMPHIBIAN\x18\x00\x00\x00-1.362885:100:BLACK SWAN\x1c\x00\x00\x00-1.366220:118:DUNGENESS CRAB\x14\x00\x00\x00-1.368406:857:THRONE\x11\x00\x00\x00-1.368889:525:DAM\x12\x00\x00\x00-1.380898:350:IBEX\x16\x00\x00\x00-1.383068:629:LIPSTICK\x17\x00\x00\x00-1.398480:995:EARTHSTAR\x15\x00\x00\x00-1.399917:597:HOLSTER\x16\x00\x00\x00-1.400566:439:BEARSKIN\x1e\x00\x00\x00-1.401573:122:AMERICAN LOBSTER\x1a\x00\x00\x00-1.412555:936:HEAD CABBAGE \x00\x00\x00-1.413827:181:BEDLINGTON TERRIER\x14\x00\x00\x00-1.415510:169:BORZOI\x16\x00\x00\x00-1.417530:476:CAROUSEL\x19\x00\x00\x00-1.418530:45:GILA MONSTER\x14\x00\x00\x00-1.422343:176:SALUKI\x12\x00\x00\x00-1.423623:746:PUCK\x1b\x00\x00\x00-1.426944:143:OYSTERCATCHER\x19\x00\x00\x00-1.427171:930:FRENCH LOAF\x16\x00\x00\x00-1.428639:228:KOMONDOR\x18\x00\x00\x00-1.444463:544:DUTCH OVEN\x1e\x00\x00\x00-1.446157:942:BUTTERNUT SQUASH\x18\x00\x00\x00-1.455556:147:GREY WHALE\x12\x00\x00\x00-1.456253:718:PIER\x1a\x00\x00\x00-1.489864:329:SEA CUCUMBER\x16\x00\x00\x00-1.490687:575:GOLFCART\x1a\x00\x00\x00-1.492030:797:SLEEPING BAG\x16\x00\x00\x00-1.494003:431:BASSINET\x19\x00\x00\x00-1.495028:565:FREIGHT CAR\x17\x00\x00\x00-1.499164:627:LIMOUSINE\x1d\x00\x00\x00-1.512210:842:SWIMMING TRUNKS\x13\x00\x00\x00-1.520539:204:LHASA\x16\x00\x00\x00-1.522396:261:KEESHOND\x18\x00\x00\x00-1.526754:441:BEER GLASS\x16\x00\x00\x00-1.530553:296:ICE BEAR\x18\x00\x00\x00-1.531576:744:PROJECTILE%\x00\x00\x00-1.531654:188:WIRE-HAIRED FOX TERRIER\x17\x00\x00\x00-1.534160:994:STINKHORN\x1f\x00\x00\x00-1.535947:821:STEEL ARCH BRIDGE\x15\x00\x00\x00-1.537152:533:DISHRAG\x17\x00\x00\x00-1.541060:601:HOOPSKIRT\x13\x00\x00\x00-1.550256:961:DOUGH\x1d\x00\x00\x00-1.555210:755:RADIO TELESCOPE\x1d\x00\x00\x00-1.558740:244:TIBETAN MASTIFF\x18\x00\x00\x00-1.563577:981:BALLPLAYER\x13\x00\x00\x00-1.571962:706:PATIO\x17\x00\x00\x00-1.578755:788:SHOE SHOP\x13\x00\x00\x00-1.593257:406:ALTAR\x13\x00\x00\x00-1.596044:982:GROOM\x18\x00\x00\x00-1.607883:175:OTTERHOUND\x17\x00\x00\x00-1.612451:121:KING CRAB\x1a\x00\x00\x00-1.620329:256:NEWFOUNDLAND\x1c\x00\x00\x00-1.620513:873:TRIUMPHAL ARCH\x12\x00\x00\x00-1.622573:99:GOOSE\x17\x00\x00\x00-1.623233:205:RETRIEVER\x14\x00\x00\x00-1.629087:927:TRIFLE\x17\x00\x00\x00-1.637138:70:HARVESTMAN\x18\x00\x00\x00-1.644518:675:MOVING VAN\x18\x00\x00\x00-1.657145:874:TROLLEYBUS\x1e\x00\x00\x00-1.674344:190:SEALYHAM TERRIER\x1a\x00\x00\x00-1.678471:39:COMMON IGUANA\x16\x00\x00\x00-1.678795:615:KNEE PAD\x1c\x00\x00\x00-1.678894:607:JACK-O-LANTERN\x1a\x00\x00\x00-1.685169:52:THUNDER SNAKE\x19\x00\x00\x00-1.686050:551:FACE POWDER\x12\x00\x00\x00-1.690842:291:LION\x14\x00\x00\x00-1.696089:471:CANNON\x1d\x00\x00\x00-1.702746:482:CASSETTE PLAYER\x1e\x00\x00\x00-1.707298:465:BULLETPROOF VEST\x1b\x00\x00\x00-1.716995:867:TRAILER TRUCK\x16\x00\x00\x00-1.720503:110:FLATWORM\x12\x00\x00\x00-1.726885:915:YURT\x15\x00\x00\x00-1.727514:866:TRACTOR\x13\x00\x00\x00-1.730903:963:PIZZA\x1f\x00\x00\x00-1.733751:540:DRILLING PLATFORM\x19\x00\x00\x00-1.735970:900:WATER TOWER\x18\x00\x00\x00-1.749231:793:SHOWER CAP\x17\x00\x00\x00-1.764530:962:MEAT LOAF\x18\x00\x00\x00-1.771308:51:TRICERATOPS"\x00\x00\x00-1.776160:757:RECREATIONAL VEHICLE\x1e\x00\x00\x00-1.785475:996:HEN-OF-THE-WOODS\x14\x00\x00\x00-1.787788:437:BEACON\x16\x00\x00\x00-1.803350:887:VESTMENT\x1b\x00\x00\x00-1.825878:123:SPINY LOBSTER\x15\x00\x00\x00-1.828053:881:UPRIGHT\x1c\x00\x00\x00-1.829382:528:DIAL TELEPHONE\x1c\x00\x00\x00-1.831217:500:CLIFF DWELLING\x17\x00\x00\x00-1.845055:479:CAR WHEEL\x14\x00\x00\x00-1.850625:717:PICKUP\x14\x00\x00\x00-1.852795:882:VACUUM\x16\x00\x00\x00-1.853920:554:FIREBOAT\x17\x00\x00\x00-1.855946:491:CHAIN SAW\x1a\x00\x00\x00-1.856789:54:HOGNOSE SNAKE\x14\x00\x00\x00-1.903447:222:KUVASZ\x19\x00\x00\x00-1.906413:511:CONVERTIBLE\x15\x00\x00\x00-1.916344:450:BOBSLED\x16\x00\x00\x00-1.950712:154:PEKINESE\x15\x00\x00\x00-1.958992:104:WALLABY\x15\x00\x00\x00-1.994070:965:BURRITO\x16\x00\x00\x00-2.002640:543:DUMBBELL"\x00\x00\x00-2.007894:229:OLD ENGLISH SHEEPDOG\x16\x00\x00\x00-2.019541:571:GAS PUMP\x18\x00\x00\x00-2.019763:351:HARTEBEEST\x12\x00\x00\x00-2.026706:914:YAWL\x16\x00\x00\x00-2.059178:685:ODOMETER\x1b\x00\x00\x00-2.068114:354:ARABIAN CAMEL\x17\x00\x00\x00-2.073867:342:WILD BOAR\x1c\x00\x00\x00-2.076504:550:ESPRESSO MAKER\x16\x00\x00\x00-2.108392:414:BACKPACK\x1a\x00\x00\x00-2.121477:770:RUNNING SHOE\x14\x00\x00\x00-2.129612:974:GEYSER\x14\x00\x00\x00-2.138828:934:HOTDOG\x16\x00\x00\x00-2.140408:939:ZUCCHINI\x13\x00\x00\x00-2.165421:347:BISON\x1f\x00\x00\x00-2.178216:28:SPOTTED SALAMANDER\x19\x00\x00\x00-2.206681:564:FOUR-POSTER\x18\x00\x00\x00-2.209337:817:SPORTS CAR\x16\x00\x00\x00-2.212073:649:MEGALITH\x14\x00\x00\x00-2.235562:106:WOMBAT\x14\x00\x00\x00-2.240308:161:BASSET\x1d\x00\x00\x00-2.241687:385:INDIAN ELEPHANT\x17\x00\x00\x00-2.244940:595:HARVESTER\x17\x00\x00\x00-2.251228:56:KING SNAKE\x14\x00\x00\x00-2.279846:529:DIAPER\x1b\x00\x00\x00-2.281298:640:MANHOLE COVER\x17\x00\x00\x00-2.295222:864:TOW TRUCK\x18\x00\x00\x00-2.297074:67:DIAMONDBACK\x16\x00\x00\x00-2.302097:561:FORKLIFT\x1e\x00\x00\x00-2.312948:217:ENGLISH SPRINGER\x17\x00\x00\x00-2.343351:833:SUBMARINE\x15\x00\x00\x00-2.354590:258:SAMOYED\x18\x00\x00\x00-2.358529:586:HALF TRACK\x19\x00\x00\x00-2.363476:153:MALTESE DOG\x13\x00\x00\x00-2.377438:340:ZEBRA\x15\x00\x00\x00-2.385032:343:WARTHOG\x12\x00\x00\x00-2.398833:847:TANK\x17\x00\x00\x00-2.408010:363:ARMADILLO\x16\x00\x00\x00-2.413244:255:LEONBERG\x15\x00\x00\x00-2.447460:657:MISSILE\x1d\x00\x00\x00-2.487332:560:FOOTBALL HELMET\x18\x00\x00\x00-2.493741:430:BASKETBALL\x18\x00\x00\x00-2.547938:766:ROTISSERIE\x13\x00\x00\x00-2.549816:884:VAULT\x16\x00\x00\x00-2.556524:625:LIFEBOAT\x14\x00\x00\x00-2.558529:29:AXOLOTL\x1a\x00\x00\x00-2.561091:160:AFGHAN HOUND\x16\x00\x00\x00-2.564135:803:SNOWPLOW\x16\x00\x00\x00-2.569652:249:MALAMUTE\x1e\x00\x00\x00-2.584881:820:STEAM LOCOMOTIVE\x1e\x00\x00\x00-2.634845:403:AIRCRAFT CARRIER\x15\x00\x00\x00-2.716956:661:MODEL T\x13\x00\x00\x00-2.761682:751:RACER\x18\x00\x00\x00-2.785010:634:LUMBERMILL\x15\x00\x00\x00-2.920391:573:GO-KART\x1c\x00\x00\x00-2.934060:194:DANDIE DINMONT\x1a\x00\x00\x00-2.964005:344:HIPPOPOTAMUS\x1a\x00\x00\x00-2.966820:933:CHEESEBURGER\x14\x00\x00\x00-2.985193:101:TUSKER\x17\x00\x00\x00-3.005315:993:GYROMITRA\x1b\x00\x00\x00-3.075230:812:SPACE SHUTTLE\x14\x00\x00\x00-3.125566:992:AGARIC\x16\x00\x00\x00-3.200765:856:THRESHER\x18\x00\x00\x00-3.284707:535:DISK BRAKE\x19\x00\x00\x00-3.409748:694:PADDLEWHEEL\x1a\x00\x00\x00-3.542693:991:CORAL FUNGUS'
_, header_length = JSONDecoder().raw_decode(response_body.decode('utf-8'))
results = httpclient.InferenceServerClient.parse_response_body(response_body, header_length=header_length)
inference_output = results.as_numpy("fc6_1").astype(str)
print(np.squeeze(inference_output)[:5])
```

output:
```
['11.547369:92:BEE EATER' '11.230166:14:INDIGO FINCH'
 '7.527154:95:JACAMAR' '6.921648:17:JAY' '6.576403:88:MACAW']
```

Reply:
created a draft PR 
https://github.com/datarobot/datarobot-user-models/pull/969/files